src/static/js/wishlist.js

```diff
@@ -91,3 +91,45 @@ setTimeout(() => {
   document.querySelectorAll('.upForm').forEach(element => { listen(element, 'up') })
   document.querySelectorAll('.downForm').forEach(element => { listen(element, 'down') })
 }, 0)
+
+function priceToFloat(price) {
+  let cleanPrice = price
+      .replace(/[^\d.,]/g, '')  // Remove everything except digits, periods, and commas
+      .replace(/,/g, '');       // Remove commas to standardize the number
```

Comment:
Christmas Community is used in places where commas are the decimal point. In Germany, you might say that something costs `5.000,10 €`. In that case, it needs to be parsed as `5000.1`.

Reply:
Maybe use some kind of logic to determine which is used last and do separate logic for them?

Reply:
Or, provide a bunch of crazy examples and have ChatGPT help? That's what I did here - it seems to work pretty well 🤣 

https://github.com/Wingysam/Christmas-Community/pull/140/commits/7c847c41e4d599b0d905d81d3b2d99d4601d7c5d